src/components/Activities/Activity3/Game/index.tsx

```diff
@@ -38,6 +47,16 @@ function Game(): JSX.Element {
     const tempVariables = storedVariables?.split(',').map(element => element as VARIABLES);
     const curVariables = tempVariables ?? [];
     setVariableSelection(curVariables);
+
+    // setup tasksSelection from storage
+    const tempTasks : number[] = storedTasks ?
+      storedTasks.split(',').map((elem) => {
+        // elem can be either '' or a number
+        return (elem && elem.length > 0) ? parseInt(elem) : 0;
+      })
+      : [0,0,0];
+    setTaskSelection(tempTasks);
+
     return () => {
       storage.removeItem(SESSION_CURRENT_STATE);
       storage.removeItem(SESSION_VARIABLES);
```

Comment:
let's remove with everyone else!
```suggestion
      storage.removeItem(SESSION_VARIABLES);
      storage.removeItem(SESSION_TASKS);
```